gef-ui/templates/_macros/submit-confirmation-title.njk

```diff
@@ -1,5 +1,9 @@
 {% macro render(params) %}
-
-  <h1 class="govuk-panel__title" data-cy="submit-confirmation-title">{{ params.submissionType | capitalize }} {{ params.text }}</h1>
-
-{% endmacro %}
+  {% if params.unissuedToIssued %}
+    <h1 class="govuk-panel__title" data-cy="submit-facilities-confirmation-title">Issued facilities
```

Comment:
{{ params.text | safe }} please to avoid XSS vulnerability.

Reply:
done